src/main/java/com/company/controller/CallController.java

```diff
@@ -0,0 +1,94 @@
+package com.company.controller;
+
+import com.company.dao.UserDAO;
+import com.google.common.collect.Lists;
+import com.twilio.jwt.client.OutgoingClientScope;
+import com.twilio.twiml.TwiMLException;
+import com.twilio.twiml.VoiceResponse;
+import com.twilio.twiml.voice.Dial;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.bind.annotation.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+import com.twilio.jwt.client.ClientCapability;
+import com.twilio.jwt.client.IncomingClientScope;
+import com.twilio.jwt.client.Scope;
+import com.twilio.twiml.voice.Client;
+
+@RestController
+@RequestMapping("/rest")
+public class CallController {
+
+    @Autowired
+    private UserDAO userDAO;
+
+    private final String ACCOUNT_SID = "ACdcacccdf505f9194fa1d718da40bfff5";
+    private final String AUTH_TOKEN = "d7120dfd13ea7e3ef63afdb714ba06af";
+    private final String applicationSid = "AP4aa3920c2cd3ac669a0dcc1a102ed3ee";
```

Comment:
1. it's best practice, when should for "final static" write uppercase font. ex: APPLICATION_SID.
2. Also, this variables. should be in table in db, in the form of constants.